mindmeld/components/entity_recognizer.py

```diff
@@ -206,6 +210,18 @@ def load(self, model_path):
         self.ready = True
         self.dirty = False
 
+    def _predict(self, query, time_zone=None, timestamp=None, dynamic_resource=None):
+        if not self._model:
+            logger.error("You must fit or load the model before running predict")
+            return None
+        if not isinstance(query, Query):
+            query = self._resource_loader.query_factory.create_query(
+                query, time_zone=time_zone, timestamp=timestamp
+            )
+        return self._model.predict(
+            [query], dynamic_resource=dynamic_resource, sys_resolver=self.sys_recognizer
```

Comment:
I dont think it makes sense to pass the recognizer to the model for every predict operation. The model should be instantiated with the system entity recognizer during construction and it should use that everytime.

Reply:
done